nixos/tests/sssd-ldap.nix

```diff
@@ -0,0 +1,93 @@
+import ./make-test-python.nix (
+  { pkgs, ... }:
+  let
+    dbDomain = "example.org";
+    dbSuffix = "dc=example,dc=org";
+
+    ldapRootUser = "admin";
+    ldapRootPassword = "foobar";
+
+    testUser = "alice";
+  in
+  {
+    name = "sssd-ldap";
+
+    meta = with pkgs.stdenv.lib.maintainers; {
+      maintainers = [ bbigras ];
+    };
+
+    nodes = {
+      server =
+        { ... }:
+        {
+          networking.firewall.allowedTCPPorts = [ 389 ];
+
+          services.openldap = {
+            enable = true;
+            rootdn = "cn=${ldapRootUser},${dbSuffix}";
+            rootpw = ldapRootPassword;
+            suffix = dbSuffix;
+            declarativeContents = ''
```

Comment:
We might need to migrate this once https://github.com/NixOS/nixpkgs/pull/94610 by @kwohlfahrt is merged.

Reply:
But not a big deal.